tsdb/db_test.go

```diff
@@ -1493,6 +1493,7 @@ func TestSizeRetention(t *testing.T) {
 		}
 	}
 	require.NoError(t, headApp.Commit())
+	db.Head().series.mmapHeadChunks(db.Head().chunkDiskMapper)
```

Comment:
It would be simpler if we just added a mmapHeadChunks method to the Head, and that does all this heavy lifting of knowing what to call and how.

Reply:
Moved it all into Head.mmapHeadChunks()